amr_core:
  ros__parameters:
    wheel_radius: 0.09
    distance_wheels: 0.660
    gear_ratio: 0.03

    enc_vel_gain: 1.0   # 측정완
    enc_wvel_gain: 1.0 # 측정완

    loop_hz: 30.0
    vx_acc: 1.0
    vy_acc: 0.5
    w_acc: 1.5